website/docs/d/nsxt_alb_service_engine_group.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_alb_service_engine_group"
+sidebar_current: "docs-vcd-datasource-nsxt-alb-service-engine-group"
+description: |-
+  Provides a data source to read NSX-T ALB Service Engine Groups. A Service Engine Group is an isolation domain that also
+  defines shared service engine properties, such as size, network access, and failover. Resources in a service engine
+  group can be used for different virtual services, depending on your tenant needs. These resources cannot be shared
+  between different service engine groups.
+---
+
+# vcd\_nsxt\_alb\_service\_engine\_group
+
+Supported in provider *v3.4+* and VCD 10.2+ with NSX-T and ALB.
+
+Provides a data source to read NSX-T ALB Service Engine Groups. A Service Engine Group is an isolation domain that also
+defines shared service engine properties, such as size, network access, and failover. Resources in a service engine
+group can be used for different virtual services, depending on your tenant needs. These resources cannot be shared
+between different service engine groups.
+
+~> Only `System Administrator` can use this data source.
+
+## Example Usage
+
+```hcl
+data "vcd_nsxt_alb_service_engine_group" "demo" {
+  name = "configured-service-engine-group"
```

Comment:
```suggestion
  name            = "configured-service-engine-group"
  sync_on_refresh = false
```

Reply:
Fair point. Changed in both - resource and data source.